### PR TITLE
Feature/logged in homepage

### PR DESCRIPTION
**Pull request by giagermain**
Logout button completed to display to the user whether or not they are logged in

### CODE REVIEW

File: evergreen/eg_app/views.py [hunk 1]
```diff
@@ -135,6 +132,7 @@ def login_view(request: HttpRequest):
         user = authenticate(request, username=email, password=password)
         if user is not None:
             login(request, user)
```
Comment (rancescolon) [inline evergreen/eg_app/views.py:134]: use `if user ` instead

**Review by rancescolon — DISMISSED** [review]

File: .env.example [hunk 2]
```diff
@@ -1,4 +0,0 @@
-POSTGRES_DB: postgres
```
Comment (WeirdAlex03) [inline .env.example]: This needs to be added back

File: evergreen/Dockerfile [hunk 3]
```diff
@@ -1,20 +1,21 @@
...
+COPY . .
+RUN ls -la /
+
+EXPOSE 8080
+
+SHELL [ "poetry", "run" ]
+
+RUN [ "python3","manage.py","makemigrations" ]
+
+# ENTRYPOINT [ "./docker-entrypoint.sh" ]
```
Comment (WeirdAlex03) [inline evergreen/Dockerfile]: Undo these changes (remove `RUN ls -la`, uncomment `ENTRYPOINT`)

File: evergreen/eg_app/views.py [hunk 4]
```diff
@@ -143,6 +140,7 @@ def login_view(request: HttpRequest):
     
     return HttpResponseBadRequest()
 
-def logout_view(request: HttpRequest):
+@csrf_exempt
+def logout_view(request: HttpRequest):
```
Comment (WeirdAlex03) [inline evergreen/eg_app/views.py]: Shouldn't need `@csrf_exempt`; remove the stray spaces added at the end of the function declaration (after the `:`)

**Review by WeirdAlex03 — CHANGES_REQUESTED** [review]

**Review by WeirdAlex03 — APPROVED** [review]
